src/middlewares/__tests__/auth.test.ts

```
@@ -0,0 +1,35 @@
+import { mockedExternalUsers } from "../../mocks/users";
+
+import express, { Request } from "express";
+
+import { createRequest } from "node-mocks-http";
+import { devAuthMiddleware } from "../auth";
+
+describe("the development authentication middleware", () => {
+  const mockedExternalUser = mockedExternalUsers[0];
+  const middleware: express.RequestHandler = devAuthMiddleware;
+
+  it("throws an error when authorization header is not set", async () => {
+    const req: Request = createRequest();
+
+    expect(() => middleware(req, null, () => null)).toThrow(Error);
```

As mentioned before, it should not return but rather return an HTTP error other than 500.